deploy/charts/jetstack-agent/Chart.yaml

```diff
@@ -0,0 +1,13 @@
+apiVersion: v2
+name: jetstack-agent
+description: Jetstack Secure Agent
```

Comment:
I'm a bit torn on whether we should still use Jestack Secure Agent, instead TLSPK Agent

Reply:
I understand that concern, but it feels wrong to change the chart name, but still have the GitHub repo as `jetstack-secure`. Perhaps we should to the renaming as a separate piece of work? It's way beyond our scope here to do that, and feel like there would be a lot of changes.

For example.. does this open-source repo move into the Venafi Organisation on GitHub as well?

Reply:
> Perhaps we should do the renaming as a separate piece of work?

I think that might be the better way to go about it :+1: I'll make a new task to cover that